### PR TITLE
Create sv.po

### PR DESCRIPTION
**Pull request by eson57**
Hi,
Here's Swedish translation if wanted.

ksnip_2020-05-04_071320

### CODE REVIEW

File: po/sv.po [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+"X-Generator: Poedit 2.0.6\n"
+"Last-Translator: Åke Engelbrektson <eson@svenskasprakfiler.se>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: sv\n"
+
+#. TRANSLATORS: Don't translate this text
+#: data/com.github.tchx84.Flatseal.desktop.in:4
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:4
+msgid "Flatseal"
+msgstr "Flatseal"
```
Comment (tchx84) [inline po/sv.po]: Please note the `#. TRANSLATORS: Don't translate this text`. You can leave it empty, e.g.

```
msgid "Flatseal"
msgstr ""
```

**Review by tchx84 — CHANGES_REQUESTED** [review]
Thanks a lot Åke, just one comment.

**Comment by tchx84** [comment]
Let me know if you need any help.

**Comment by eson57** [comment]
Done!
Sorry, I don't see that comment using PoEdit and I also don't see how it matters as long as I do not translate.

**Review by tchx84 — APPROVED** [review]

**Comment by tchx84** [comment]
Thanks Åke!

**Pull request merged**